Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -98,6 +98,10 @@ namespace AzToolsFramework
             //! @param context reflection context.
             static void Reflect(AZ::ReflectContext* context);
 
+            //! Whether caching the instance's dom should be enabled by default.
```

Comment:
do you mean "disabled"?

Reply:
Well the variable name is in the positive, so I think the comment makes sense?

Reply:
Not sure if I understand it right. What it is doing here is have a static variable that represents the default behavior of whether we want to cache the instance DOM or not. And we don't want it to cache the instance DOM by default. Is it correct?

Reply:
Correct. The comment states that the variable determines if the dom should be cached.

Reply:
okay I see. No problem.

My previous interpretation:
(Whether caching the instance's dom) should be enabled by default.

What it actually indicates:
Whether (caching the instance's dom should be enabled by default).